core/src/main/scala/com/rallyhealth/vapors/factfilter/data/Evidence.scala

```diff
@@ -0,0 +1,73 @@
+package com.rallyhealth.vapors.factfilter.data
+
+import cats.{Monoid, Order}
+import cats.data.NonEmptySet
+import cats.instances.order._
+
+import scala.collection.immutable.SortedSet
+
+final class Evidence private (val factSet: SortedSet[Fact]) extends AnyVal {
+
+  def isEmpty: Boolean = factSet.isEmpty
+  def nonEmpty: Boolean = factSet.nonEmpty
```

Comment:
I'm generally against any negatively phrased boolean concepts, since they nearly always immediately lead to double negation, and no humans are not bad at understanding that.

Reply:
I thought the point of a `nonEmpty` method was to avoid a double negative at the call-site... So you can write `if (x.nonEmpty)` versus `if (!x.isEmpty)`